### ISSUE TITLE
V3 Shop error

### ISSUE THREAD

**Issue by causefx** [Redjumpman/Jumper-Cogs#156]
When running the command `shop buy` I get presented with the shop list... once I select which list, i.e. `1` I get hit with the error:

![image](https://user-images.githubusercontent.com/16184466/46574883-af01b180-c95f-11e8-88eb-7fa00c39c720.png)

Side note:
Buying the item directly `shop buy STORE ITEM` works.

```
[06/10/2018 18:59] ERROR events on_command_error 211: Exception in command 'shop buy'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 61, in wrapped
    ret = await coro(*args, **kwargs)
  File "/data/cogs/CogManager/cogs/shop/shop.py", line 168, in buy
    shop, item = await menu.display()
  File "/data/cogs/CogManager/cogs/shop/menu.py", line 21, in display
    item = await self.menu_loop(data, groups, page, maximum, msg)
  File "/data/cogs/CogManager/cogs/shop/menu.py", line 63, in menu_loop
    item = await self.next_menu(data, selection, msg)
  File "/data/cogs/CogManager/cogs/shop/menu.py", line 191, in next_menu
    msg, groups, page, maximum = await self.setup(data=new_data, msg=msg)
  File "/data/cogs/CogManager/cogs/shop/menu.py", line 44, in setup
    e = await self.build_menu(groups, page)
  File "/data/cogs/CogManager/cogs/shop/menu.py", line 129, in build_menu
    line_one = (f"{f'{idx}.': <{3}} {x[0]: <{28}s} {x[1]['Qty']: < {8}}"
ValueError: Sign not allowed in string format specifier
```

**Comment by Redjumpman** [comment]
Just pushed an update that will fix that in 3.1.04.

**Comment by causefx** [comment]
confirmed to work, thank you.
